lib/components/Dialog/Dialog.docs.tsx

```diff
@@ -0,0 +1,375 @@
+import React, { useState, Fragment } from 'react';
+import { ComponentDocs } from '../../../site/src/types';
+import {
+  Actions,
+  Dialog,
+  Button,
+  Inline,
+  Text,
+  Stack,
+  TextLink,
+  IconMail,
+  Box,
+  Strong,
+} from '../';
+import {
+  Placeholder,
+  Dialog as PlayroomDialog,
+} from '../../playroom/components';
+
+const docs: ComponentDocs = {
+  category: 'Layout',
+  screenshotWidths: [320],
```

Comment:
Is there any point screenshotting this?

Reply:
Im abstracting out a DialogCard internally and going to screenshot that. Its too important to not screenshot, but these in their current state are of no use.